sys-kernel/bliss-kernel-bin/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person">
+        <email>jon@xyinn.org</email>
```

Comment:
as I have said, switch to your gento org addrr.

Reply:
fixing